src/temba/TembaSelect.tsx

```
@@ -46,6 +46,9 @@ export interface TembaSelectProps {
   clearable?: boolean;
 
   queryParam?: string;
+
+  maxItems?: Number;
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)